common/arg.cpp

```diff
@@ -2466,7 +2466,7 @@ common_params_context common_params_parser_init(common_params & params, llama_ex
     ).set_examples({LLAMA_EXAMPLE_SPECULATIVE, LLAMA_EXAMPLE_SERVER}).set_env("LLAMA_ARG_N_CPU_MOE_DRAFT"));
     add_opt(common_arg(
         {"-ngl", "--gpu-layers", "--n-gpu-layers"}, "N",
-        "number of layers to store in VRAM",
+        string_format("number of layers to store in VRAM (default: %d, 999 = max layers)", params.n_gpu_layers),
```

Comment:
I'm not happy with this wording w.r.t. "999 = max layers". Maybe "max. number of model layers to store in VRAM"?